base_station/gui/src/components/Chlorophyll.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+<div>
+  <div class="buttons">
+    <div>
+      <h3> Chlorophyll Data</h3>
+    </div>
+
+    <label for="toggle_button" :class="{'active': white_ledS == 1}" class="toggle__button">

```

Comment:
why is this white_ledS instead of white_leds?

Reply:
That's an execellent question. Guthrie redid some of the naming of mosfet commands today and this may have gotten changed but still embarrassing